applications/portfolioDrafts/utils/commands.ts

```diff
@@ -0,0 +1,27 @@
+import { UpdateCommand } from "@aws-sdk/lib-dynamodb";
+import { PortfolioStep } from "../models/PortfolioStep";
+
+export function portfolioStepCommand(
+  table: string | undefined,
```

Comment:
This doesn't handle undefined, it'd be invalid to pass undefined to the API, and so we should not allow undefined as a type.